CMakeLists.txt

```diff
@@ -508,9 +514,10 @@ if(NOT USE_SYSTEM_MINIUPNPC)
         INTERFACE
             MINIUPNP_STATICLIB)
 
-    set(MINIUPNPC_VERSION 1.9)
-    set(MINIUPNPC_API_VERSION 12)
+    set(MINIUPNPC_VERSION 2.2)
+    set(MINIUPNPC_API_VERSION 17)
```

Comment:
You can consider cherry picking my changes from #6665 and drop these 2 lines completely.

Reply:
I'll leave it in the hands of your PR: there are other lines related to MINIUPNPC_VERSION, like 7 lines below, and I'm not sure of the consequences of removing those `set` but leaving the `$<$<VERSION_LESS:${MINIUPNPC_VERSION},1.7>:MINIUPNPC_API_VERSION=${MINIUPNPC_API_VERSION}>`.

Reply:
Sure thing. Just fyi, these variables are already set at that point, you can see the description from my PR for a more detailed explanation.